### PR TITLE
Add blogger 1

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- There are no news/blog possibilities. #20 

**What is the new behavior (if this is a feature change)?**
- A news page, including a search page and article page has been added. #20

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+    "editor.fontFamily": "Fira Code, Consolas, 'Courier New', monospace",
+    "editor.fontLigatures": true,
+    "editor.snippetSuggestions": "top",
+    "terminal.integrated.fontFamily": "Consolas",
+    "workbench.colorTheme": "Dracula Soft",
+    "cSpell.ignoreWords": [
+        "aichner",
```
Comment (kleberbaum) [inline .vscode/settings.json:8]: Is it necessary to extend the editor env settings?

File: .vscode/extensions.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+    "recommendations": [
```
Comment (kleberbaum) [inline .vscode/extensions.json:2]: Is it necessary to extend the editor env settings?

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

File: .vscode/settings.json (hunk 1 above)
Comment (Aichnerc) [reply .vscode/settings.json:8]: Yes it is.

File: .vscode/extensions.json (hunk 2 above)
Comment (Aichnerc) [reply .vscode/extensions.json:2]: Yes it is. Those are the tools that are best to use for development in a React.js environment.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**